src/_includes/docs/debug/debug-flow-vscode-as-start.md

```diff
@@ -11,7 +11,7 @@
    **View** <span aria-label="and then">></span>
    **Command Palette...**
 
-   You can also press <kbd>Command</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>.
+   You can also press <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>.
```

Comment:
I know you didn't touch this here, but looks like this should also have a non-macos key binding.